### PR TITLE
PR with schema, core and makefile changes

### PR DESCRIPTION
**Pull request by sunitakawane**

### CODE REVIEW

File: src/juxt/jsonschema/core.cljc [hunk 1]
```diff
@@ -20,7 +20,18 @@
 (defn schema? [x]
   (or (object? x) (boolean? x)))
 
+
 (defn regex? [x]
```
Comment (shivekkhurana) [inline src/juxt/jsonschema/core.cljc:24]: `regex?` should return a boolean. The java version returns Pattern or map.

File: deps.edn [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
+  :test {:extra-paths ["test"]
+         :extra-deps {com.cognitect/test-runner {:git/url "https://github.com/cognitect-labs/test-runner.git"
+                                                 :sha "028a6d41ac9ac5d5c405dfc38e4da6b4cc1255d5"}}
+         :main-opts ["-m" "cognitect.test-runner"]}
+
+  :dev-nrepl {:jvm-opts ["-Dnrepl.load=true"]
+              :extra-paths ["aliases/nrepl"]
+              :extra-deps
+              {cider/cider-nrepl {:mvn/version "0.17.0"}
+               ;;refactor-nrepl {:mvn/version "2.3.1"}
```
Comment (shivekkhurana) [inline deps.edn]: Can get rid of this dep.

File: Makefile [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
-	clj -Atest -e :deprecated
+STYLESDIR = ../asciidoctor-stylesheet-factory/stylesheets
+STYLESHEET = juxt.css
+
+.PHONY: 		watch default deploy test 
 
 official-test:
 	clj -Atest -i :official
 
+test-clj:
```
Comment (shivekkhurana) [inline Makefile:11]: Tabs seem to be out of place. Use only a single tab (like `official-test`)

**Review by shivekkhurana — CHANGES_REQUESTED** [review]

**Review by shivekkhurana — APPROVED** [review]

**Review by malcolmsparks — APPROVED** [review]
Yes, all good. Thanks

**Pull request merged**